src/Surfnet/StepupSelfService/SelfServiceBundle/Controller/RemoteVettingController.php

```diff
@@ -141,27 +139,20 @@ public function acsAction(Request $request)
         $this->logger->info('Load the attributes from the saml response');
 
         try {
-            $processId = $this->samlCalloutHelper->handleResponse($request, 'IRMA');
+            $processId = $this->samlCalloutHelper->handleResponse($request, $this->remoteVettingService->getActiveIdentityProviderSlug());
         } catch (InvalidRemoteVettingStateException $e) {
             $this->logger->error($e->getMessage());
             $flashBag->add('error', 'ss.second_factor.revoke.alert.remote_vetting_failed');
             return $this->redirectToRoute('ss_second_factor_list');
-        } catch (Exception $e) {
-            //PreconditionNotMetException $e) {
-            // todo: add saml specific logging (user flashbag may differ) ?
-            $this->logger->error($e->getMessage());
+        } catch (PreconditionNotMetException $e) {
+            $this->logger->notice($e->getMessage());
```

Comment:
You might want to raise the log level a notch? This is quite serious

Reply:
That depends, it could also be just a user that cancelled. Will change it to `error` for now and I'll add a task to the ticket below. This maybe something IdP specific.
`https://www.pivotaltracker.com/story/show/171874937`

Reply:
Good to task this on that ticket